Лабораторная работа №1
по дисциплине «Анализ данных».

> Задание:

Найти датасет.
* Произвести выборки,
* определить t-критерий Стьюдента.
* anova test
* Сделать выводы.

* H0 - мячики с покрытием летят так же далеко, как и без него
* H1 - мячики с покрытием летят дальше, чем без него


In [ ]:
import numpy as np
import scipy.stats as stats
import matplotlib.pyplot as plt
import csv

%matplotlib inline

In [ ]:
with open('Golf.csv', 'r', newline='') as csvfile:
    data = csv.reader(csvfile, delimiter=',')
    a1 = []
    a2 = []
    next(data)
    for row in data:
        a1.append(int(row[1]))
        a2.append(int(row[0]))

In [ ]:
rez = stats.ttest_ind (a=a1, b=a2)
print(rez)

Ttest_indResult(statistic=-1.3283615935245678, pvalue=0.18793228491854663)


#### т.к. значение р-теста (0.1879) больше, чем альфа = 0,05, следовательно, на уровне статистики мы не можем отвергнуть нулевую гипотезу и утверждать что мячики с покрытием летят дальше, чем без него, нужны дополнительные исследования

## ANOVA
Построим блочную диаграмму:

In [ ]:
from scipy.stats import f_oneway

f_oneway(a1, a2)

F_onewayResult(statistic=1.764544523151157, pvalue=0.18793228491854264)

P > 0.05, поэтому не можем отклонить нулевую гипотезу.